src/app/components/arlas-wui-root/arlas-wui-root.component.ts

```diff
@@ -1245,6 +1305,38 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
           }
         }
         break;
+      case 'production':
+        this.processService.load().subscribe({
```

Comment:
There is a duplication of this code block. It could be put in a method that takes a list of strings as an input, so either `idsItemSelected.map(i => i.idValue)` or `[data.elementidentifier.idValue]` to avoid duplication